Makefile

```diff
@@ -8,16 +8,14 @@ PYTHON=python
 all: build
 
 add-submodules:
-	-git submodule add -b v0.8.0 https://github.com/alanxz/rabbitmq-c.git
+	git submodule add -b master https://github.com/alanxz/rabbitmq-c.git
```

Comment:
I'm very not comfortable with this change.
You may bump the library version to 0.9.0.

Always building from master can cause breakage.

Reply:
I want to pull in a few fixes post 0.9.0 that are upstream , maybe i can use a commit hash ?

Reply:
That'd be much better.
Alternatively, ask for a new upstream release.